eng/helix/content/RunTests/ProcessUtil.cs

```diff
@@ -171,17 +170,17 @@ public static Task CaptureDumpAsync(int pid, string dumpFilePath)
             process.BeginErrorReadLine();
 
             var canceledTcs = new TaskCompletionSource<object?>();
-            await using var _ = cancellationToken.Register(() => canceledTcs.TrySetResult(null));
+            await using var _ = cancellationToken.Register(() => canceledTcs.TrySetResult(null)).ConfigureAwait(false);
```

Comment:
I suggested adding `ConfigureAwait(...)` only because I thought the number of cases was low. Seeing this, I prefer adding `CA2007`. That's necessary because the project targets `net7.0` and not anything `netstandard`